src/components/SwapWidget/LimitOrder/index.tsx

```diff
@@ -79,51 +79,57 @@ const LimitOrder: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisibl
   const inputTokenInfo = gelatoInputCurrency?.tokenInfo;
   const outputTokenInfo = gelatoOutputCurrency?.tokenInfo;
 
-  const inputCurrency =
-    gelatoInputCurrency && gelatoInputCurrency?.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : inputTokenInfo && inputTokenInfo.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : inputTokenInfo
-      ? new Token(
-          inputTokenInfo?.chainId,
-          inputTokenInfo?.address,
-          inputTokenInfo?.decimals,
-          inputTokenInfo?.symbol,
-          inputTokenInfo?.name,
-        )
-      : gelatoInputCurrency && gelatoInputCurrency.isToken
-      ? new Token(
-          gelatoInputCurrency?.chainId,
-          gelatoInputCurrency?.address,
-          gelatoInputCurrency?.decimals,
-          gelatoInputCurrency?.symbol,
-          gelatoInputCurrency?.name,
-        )
-      : undefined;
-
-  const outputCurrency =
-    gelatoOutputCurrency && gelatoOutputCurrency?.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : outputTokenInfo && outputTokenInfo?.symbol === CAVAX[chainId].symbol
-      ? CAVAX[chainId]
-      : outputTokenInfo
-      ? new Token(
-          outputTokenInfo?.chainId,
-          outputTokenInfo?.address,
-          outputTokenInfo?.decimals,
-          outputTokenInfo?.symbol,
-          outputTokenInfo?.name,
-        )
-      : gelatoOutputCurrency && gelatoOutputCurrency.isToken
-      ? new Token(
-          gelatoOutputCurrency?.chainId,
-          gelatoOutputCurrency?.address,
-          gelatoOutputCurrency?.decimals,
-          gelatoOutputCurrency?.symbol,
-          gelatoOutputCurrency?.name,
-        )
-      : undefined;
+  const inputCurrency = () => {
```

Comment:
please use proper naming convention

Reply:
This name is not decided by me. I just reused it and fixed sonarlint error.
Please have a look at diff.